nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/test/resources/override-tag-service-properties.xml

```diff
@@ -4,7 +4,7 @@
   <require>org.nuxeo.tag.service.properties</require>
 
   <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
-    <property name="nuxeo.faceted.tag.service.enabled">false</property>
+    <property name="nuxeo.faceted.tag.service.enabled">true</property>
```

Comment:
I would name this file more like "faceted-tag-service-override.xml" to be clear.